ib_manifest_util/cli.py

```diff
@@ -42,13 +42,18 @@ def main() -> None:
     "--output_dockerfile_path",
     help="(Optional) Path to location in which the Dockerfile will be placed",
 )
+@click.option(
```

Comment:
It's not clear why I would need a virtual package spec. I know you explained this and maybe it would be apparent by the time I'm the person using this tool to update an IB repo. How do I know the schema of the yaml to create? Is there a function to export the default package spec? Would it be useful if there was a suboption that output the default spec in the current user environment?

Reply:
this all exists in conda-vendor.  this is essentially a pass-through option.  i don't really think we need it for ib_manifest_util.